src/pages/workspaces/workspace/analysis/modals/ComputeModal.test.js

```diff
@@ -619,29 +617,26 @@ describe('ComputeModal', () => {
     })
 
     // Assert
-    expect(screen.getByText(`${tools.Jupyter.label} Cloud Environment`)).toBeInTheDocument()
+    screen.getByText(`${tools.Jupyter.label} Cloud Environment`)
 
     const selectText = hailImage.label
-    expect(screen.getByText(selectText)).toBeInTheDocument()
+    screen.getByText(selectText)
 
-    expect(screen.getByText(machine1.cpu)).toBeInTheDocument()
-    expect(screen.getByText(machine1.memory)).toBeInTheDocument()
-    expect(screen.getByText('Spark cluster')).toBeInTheDocument()
+    screen.getByText(machine1.cpu)
+    screen.getByText(machine1.memory)
+    screen.getByText('Spark cluster')
 
     verifyDisabled(screen.getByLabelText('Workers'))
     verifyDisabled(screen.getByLabelText('Location'))
 
-    //TODO: fix bug affecting this, IA-3739
-    //master and worker disk sizes, there is a bug currently
-    // expect(screen.getByDisplayValue(150)).toBeInTheDocument()
-    // expect(screen.getByDisplayValue(151)).toBeInTheDocument()
+    screen.getByDisplayValue(150)
+    screen.getByDisplayValue(151)
```

Comment:
These are values in an input element? If so, I think it would be clearer to get the input element and check its value instead of asserting that these numbers appear somewhere on the page.

Reply:
Ah true. Issue was both elements were identical with the bug before. Now that the bug is fixed I can do a `getAll` and assert there are 2 elements with the different values.